src/nvim/testdir/test_signs.vim

```diff
@@ -1143,7 +1153,7 @@ func Test_sign_id_autogen()
   call writefile(repeat(["Sun is shining"], 30), "Xsign")
   edit Xsign
 
-  call assert_equal(1, sign_place(0, '', 'sign1', 'Xsign',
+  call assert_equal(2, sign_place(0, '', 'sign1', 'Xsign',
```

Comment:
Bug?

Reply:
Still need to investigate whether this is ok. As it is only an id I don't think it needs to be identical across versions of the code, it just need to be unique.